config/default/core.entity_form_display.node.agency_component.default.yml

```diff
@@ -17,8 +17,8 @@ dependencies:
     - field.field.node.agency_component.field_request_submission_form
     - field.field.node.agency_component.field_service_centers
     - field.field.node.agency_component.field_submission_address
-    - field.field.node.agency_component.field_submission_api
     - field.field.node.agency_component.field_submission_api_secret
+    - field.field.node.agency_component.field_submission_api_url
```

Comment:
What're your thoughts on retaining the machine name `field_submission_api`? If we proceed with changing it to `field_submission_api_url`, we need to be sure we're modifying references to the field that explicitly look for `field_submission_api`. I know jsonapi config has references to that, I'm surprised it didn't get included in your exported config. Also, the api code coming down the pipeline looks for it as well.